includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -1750,13 +1751,19 @@ public function add_status_change_note( $order, $charge_permission_id, $new_stat
 	 * @param string $id The value to provide the id attribute of the script with.
 	 * @return void
 	 */
-	public function update_js( $id = 'wc-apa-update-vals' ) {
+	public function update_js( $id ) {
```

Comment:
```suggestion
	public function update_js( $id = '') {
```
To keep compatibility in the case this function was called with out parameter